repair/featurize/currentsimfeat.py

```diff
@@ -9,33 +9,39 @@
 def gen_feat_tensor(input, classes, total_attrs):
     vid = int(input[0])
     attr_idx = input[1]
-    init_value = input[2]
+    current_value = input[2]
+    domain = input[3].split('|||')
     # TODO: To add more similarity metrics increase the last dimension of tensor.
     tensor = torch.zeros(1, classes, total_attrs)
-    domain = input[2].split('|||')
```

Comment:
This is the bug where we did not actually use the `domain` values but rather just the `init_value`, so it would always set `sim` to `-1.0` for `idx = 0` (which does not actually correspond to the domain value at index `0` but rather the `init_value`).

Reply:
Thanks for catching this.